md/hardware-and-software-requirements.md

```diff
@@ -17,57 +17,38 @@ Hardware required for Bonita BPM Platform (Bonita BPM Engine and Bonita BPM Port
-| Tomcat 7.0.55 | ✔ | | | |
-| Tomcat 7.0.67 |  | ✔ | ✔ | ✔ |
-| JBoss AS 7.1.1  |✔ | ✔ | | |
-| WildFly 10.1.0  | | | ✔ | ✔ |
-| Weblogic server 12c (12.1.\*) | ✔ | ✔ | ✔ | |
+| Apache Tomcat | 7.0.x (x > 67) |
+| Red Hat WildFly | 10.1.x |
 | **Database** |
-| MySQL 5.5.27 (see notes 1 and 2) | ✔ | ✔ | ✔ | ? |
-| PostgreSQL 9.3  |✔ | ✔ | ✔ | ✔ |
-| SQL Server 2012 (see note 3) | ✔ | ✔ | ✔ | ✔ |
-| Oracle 11gR2 (11.2.0.x) | ✔ | ✔ | ✔ | ✔ |
-| Oracle 12c (12.1.0.x.y) | | ✔ | ✔ | ✔ |
+| MySQL | 5.5.27 and higher (see note 1) |
+| PostgreSQL | 9.3 and higher |
+| SQL Server | 2012 (see note 2) |
+| Oracle  | 11gR2 (11.2.0.x) and 12c (12.1.0.x.y) |
 | **Browser** |
-| Internet Explorer 9.x and newer | ✔ | ✔ | ✔ | |
-| Firefox (2 last versions) | ✔ | ✔ | ✔ | ✔ |
-| Chrome (2 last versions) | ✔ | ✔ | ✔ | ✔ |
-| Microsoft Edge | | | | ✔ | 
+| Mozilla Firefox | latest version |
+| Google Chrome | latest version |
+| Microsoft Edge | latest version |
 | **Mobile** |
-| iOS 7.1 -- Safari -- Approved on Apple iPhone 5 | ✔ | ✔ | ✔ | ✔ |
-| Android 4.4 -- Chrome -- Approved on Nexus 5 | ✔ | ✔ | ✔ | ✔ |
+| Mozilla Firefox | latest version |
+| Google Chrome | latest version |
+| Apple Safari | latest version |
 
 Notes:
 
 1. MySQL must be configured with innoDB storage engine.
-2. **Warning (7.0.x only, fixed starting 7.1.0) :** There are known issues with the management of XA transactions by MySQL engine and driver: see MySQL bugs [17343](http://bugs.mysql.com/bug.php?id=17343) and [12161](http://bugs.mysql.com/bug.php?id=12161) for more details.
-Thus, using MySQL database in a production environment does not work and is not supported.
-3. **Warning:** There is a known issue between Bitronix (the Transaction Manager shipped by Bonitasoft for the Tomcat bundle and inside the Deploy bundle for Tomcat) and Microsoft SQL Server driver
+2. There is a known issue between Bitronix (the transaction manager shipped by Bonitasoft for the Tomcat bundle and inside the Deploy bundle for Tomcat) and Microsoft SQL Server JDBC driver
 (refer to: [MSDN note](https://msdn.microsoft.com/en-us/library/aa342335.aspx), [Bitronix note](http://bitronix-transaction-manager.10986.n7.nabble.com/Failed-to-recover-SQL-Server-Restart-td148.html)).
-Therefore, using Bitronix as a Transaction Manager with SQL Server does not work and is not supported. To use SQL Server database requires that you use the JBoss or WildFly bundle provided by Bonitasoft.
-4. Bonita BPM can be executed on Java 8 and above. All development artifacts (connectors, REST API extensions, etc) must be compiled with Java 8 byte code (target version).
+Therefore, using Bitronix as a transaction manager with SQL Server does not work and is not supported. To use SQL Server database requires that you use the WildFly bundle provided by Bonitasoft.
+3. Bonita BPM can be executed on Java 8 and above. All development artifacts (connectors, REST API extensions, etc) must be compiled with Java 8 byte code (target version).
```

Comment:
No need to re-number all bullet points, as soon as they are numbers.
This avoids false-positive potential conflicts when merging.